.github/workflows/close-stale-feature-requests.yml

```diff
@@ -0,0 +1,43 @@
+name: Close stale feature requests
+on:
+  schedule:
+    - cron: 0 1 * * *
+
+env:
+  CLOSE_MESSAGE: >
+    There has been no activity on this feature request
+    and it is being closed. We value your input
+    and if you feel closing this issue was not the
+    right thing to do, please re-open it.
+
+    For more information on how the project manages
+    feature requests please consult the
+    [feature request management document](https://github.com/nodejs/node/blob/HEAD/doc/guides/feature-request-management.md).
+
+  WARN_MESSAGE: >
+    There has been no activity on this feature request for
+    5 months and it is unlikely to be implemented.
+    It will be closed after 6 months after the last non-automated comment
+    unless there is renewed discussion or collaborators advocating that
+    it be kept open. For more information on how the project manages
+    feature requests please consult the
+    [feature request management document](https://github.com/nodejs/node/blob/HEAD/doc/guides/feature-request-management.md).
+
+jobs:
+  stale:
+    if: github.repository == 'nodejs/node'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 150
+          days-before-close: 30
+          stale-issue-label: stale
+          close-issue-message: ${{ env.CLOSE_MESSAGE }}
+          stale-issue-message: ${{ env.WARN_MESSAGE }}
+          only-labels: feature request
+          exempt-pr-labels: never-stale
+          # max requests it will send per run to the GitHub API before it deliberately exits to avoid hitting API rate limits
+          operations-per-run: 1
```

Comment:
I've set this to 1 so that it only processes 1 per day to start so that we can verify it's working as expected before a larger number of feature requests get tagged.

If there is a better way to test this please let me know. Otherwise the plan would to see it run a few days to make sure it operates as expected and then ramp up to a larger number. Probably still lower than the max (maybe 10 per day) to avoid a flood of stale feature requests all at the same time.

@phillipj I see you added a similar one to close stalled issues. If you have any suggestions/recommendations for testing/roll out they would be greatly appreciated.

Reply:
There is also [`debug-only`](https://github.com/actions/stale#debug-only) option.

Reply:
@Mesteery would you suggest the debug-only option as the best way to test out. Looking for any experience/suggestions based on how we tested in the past.

Reply:
> would you suggest the debug-only option as the best way to test out

No, it was just to inform. Personally, I find `operations-per-run` a good way to test out.

Reply:
@Mesteery thanks for the confirmation